doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -87,42 +98,41 @@ The following client-side profile is based on the rules that sets schema.
-be opened from the application menu on the navigation panel as illustrated by the following screenshot.
+### Setting client-side EP profile schema from Administration UI
+
+To view the list of client-side endpoint profile schemas created by a [tenant developer]({{root_url}}Glossary/#tenant-developer), open the the **Client-side EP profile** page under the **Schemas** section of the application.
 
 <img src="admin-ui/Client-side endpoint profile schema.png">
 
-To export the client-side EP profile schema, click **Export** in the last column of desired schema row and select export method from drop-down. 
-See [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
-As a tenant developer, you can create new client-side EP schemas for the application as follows:
+To export a client-side EP profile schema, click the corresponding **Export** button and choose the export method from the drop-down list.
+For more information about the schema export methods, see [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support).
+
+As a tenant developer, you can create new client-side EP schemas for your application as follows:
+
+1. Under the **Schemas** section of the application, click **Client-side EP profile schemas**, then click the **Add schema** button.
+
+2. On the **Add profile schema** page, enter your schema name and description (optional).	
 
-1. In the **Client-side EP profile schemas** window for the application, click **Add schema**.
-2. In the **Add profile schema** window enter the name of the schema.
 3. Then create a schema using one of the two options:
-    * Using the existing CT by clicking **Select existing type** and selecting exiting CT version from FQN and version drop-downs.
+
+    * Use the existing [common type (CT)]({{root_url}}Glossary/#common-type-ct) by clicking **Select existing type**.
+    Click the **Select fully qualified name of existing type** field and select the CT from the drop-down list, then select the version from the corresponding drop-down list.
     
     <img src="admin-ui/Create client-side endpoint profile schema 1.png">
     
-    * Create new CT by clicking **Create new type**. In this case you will be redirected to **Add new type** window. Here you can create a schema either by using 
-    the schema form or by uploading a schema in the [Avro](http://avro.apache.org/docs/current/spec.html) format from a file.
+    * Create new CT by clicking **Create new type**.
+    The **Add new type** page will open.
+    Here you can create a schema either by using the schema form or by uploading a file containing the schema in the [Avro](http://avro.apache.org/docs/current/spec.html) format.
```

Comment:
We do not support 100% the Avro format, some features are not supported, and some extra attributes are defined. I suppose that a separate subsection has to cover the schema format in the CTL page, and you should link to that from here.